usaspending_api/config/envs/production.py

```diff
@@ -0,0 +1,29 @@
+########################################################################################################################
+# Production runtime env configuration
+# - Inherits all defaults from DefaultConfig in default.py
+# - Overrides any default config variables with local runtime env specific values
+# - Users may override local config vars values declared here by putting values in a git-ignored .env file alongside
+#   docker-compose.yml
+# - Set config variables to DefaultConfig.USER_SPECIFIC_OVERRIDE where there is expected to be a
+#   user-provided a config value for a variable (e.g. in the ../.env file)
+########################################################################################################################
+from typing import ClassVar
+
+from usaspending_api.config.envs.default import DefaultConfig
+
+
+class ProductionConfig(DefaultConfig):
+    """Config for a production runtime environment, which inherits and overrides from DefaultConfig
+
+    See Also:
+        Attributes inherited from or overridden from ``DefaultConfig``
+
+    """
+
+    # ==== [Global] ====
+    ENV_CODE: ClassVar[str] = "prd"
+
+    # ==== [AWS] ====
+    AWS_PROFILE: str = None
+    AWS_REGION = ""
```

Comment:
Only thing I'm not sure of is why we have
```
AWS_REGION = ""
```
I would think we'd want that set, and it could be set in the `DefaultConfig` and not needing overriding in lower envs

And if we remove it, probably remove it from each qat.py, staging.py, production.py
If we keep it, I think it needs to be typed like it is in `DefaultConfig`

```
AWS_REGION: str = ""
```